esp_encrypted_img/test/test.c

```diff
@@ -321,7 +321,8 @@ TEST_CASE("Test canceling decryption frees memory", "[encrypted_img]")
     esp_decrypt_handle_t ctx = esp_encrypted_img_decrypt_start(&cfg);
     TEST_ASSERT_NOT_NULL(ctx);
 
-    (void) esp_encrypted_img_decrypt_end(ctx);
```

Comment:
Shall we use newly added API in the tests or example code?

Reply:
I have added test cases that uses the newly added APIs